app/src/main/res/values-en/strings.xml

```diff
@@ -925,24 +925,24 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_kerb_height_lowered_ramp">Curb ramp</string>
     <string name="quest_kerb_height_no">No curb at all</string>
 
-    <string name="quest_lanes_title">"How many lanes for cars does this road have?"</string>
+    <string name="quest_lanes_title">"How many car lanes does this road have?"</string>
     <string name="quest_lanes_answer_lanes">"Marked lanes"</string>
-    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs for each side"</string>
-    <string name="quest_lanes_answer_lanes_description2">"Specify the total number of marked lanes (both directions) usable by car traffic. So, bike and dedicated parking lanes don’t count.
+    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs on each side"</string>
+    <string name="quest_lanes_answer_lanes_description2">"Specify the total number of marked lanes (both directions) usable by car traffic. Bike and dedicated parking lanes don’t count.
 If any lanes are reserved for buses, please leave a note instead."</string>
-    <string name="quest_lanes_answer_lanes_description_one_side2">"Specify the number of marked lanes usable by car traffic here. So, bike and dedicated parking lanes don’t count.
+    <string name="quest_lanes_answer_lanes_description_one_side2">"Specify the number of marked lanes usable by car traffic here. Bike and dedicated parking lanes don’t count.
 If any lanes are reserved for buses, please leave a note instead."</string>
     <string name="quest_lanes_answer_noLanes">"No marked lanes"</string>
     <string name="quest_lanes_answer_lanes_center_left_turn_lane">"It has a two-way left turn lane"</string>
 
-    <string name="quest_laundrySelfService_title2">Does this laundry provide self-service?</string>
+    <string name="quest_laundrySelfService_title2">Is this a self-service laundromat?</string>
 
     <string name="quest_leafType_title">Do the trees here have needles or leaves?</string>
     <string name="quest_leaf_type_needles">Needles</string>
-    <string name="quest_leaf_type_broadleaved">Broadleaved</string>
+    <string name="quest_leaf_type_broadleaved">Leaves</string>
     <string name="quest_leaf_type_mixed">Both are present</string>
 
-    <string name="quest_level_title2">On which level number is this place located?</string>
+    <string name="quest_level_title2">On which floor is this place located?</string>
```

Comment:
For example the "Ground" floor may be "G" but this quest is strictly asking for a numeric value to order floors.

Reply:
Thanks for clarifying this. I asked a similar question in POEditor because Vietnamese has two different words that imply a different starting number.

In American English, “level number” and “floor number” are synonymous and interchangeable, so if there’s anything else you wanted to communicate about this question, it’ll have to be more explicit.